runtime/doc/vim9class.txt

```diff
@@ -749,7 +749,9 @@ type alias.  For Example: >
 	:type ListOfStrings = list<string>
 
 The type alias can be used wherever a built-in type can be used.  The type
-alias name must start with an upper case character.
+alias name must start with an upper case character.  A type alias can be
+created only at the script level and not inside a function.  A type alias can
+be exported and used across scripts.
```

Comment:
While it's true that "...used wherever a built-in type can be used." To avoid confusion, might want sentence about the limitations of how a type alias works with user defined types. In particular, a type alias of a class cannot be used wherever a **type** can be used.

For example
```
vim9script
class C
endclass
type Ctype = C
echo Ctype.new()
```

Reply:
What about changing that sentence to "..used wherever a built-in type is used to specify a variable type"?

Reply:
The existing wording is accurate.

My confusion came from not being able to use the type alias as the type; the behavior didn't meet my expectations, so I missed the implications when reading the help.